Src/WitsmlExplorer.Api/Services/RigService.cs

```diff
@@ -47,18 +64,30 @@ public async Task<Rig> GetRig(string wellUid, string wellboreUid, string rigUid)
 
             return new Rig
             {
-                AirGap = witsmlRig.AirGap,
-                DateTimeCreation = StringHelpers.ToDateTime(witsmlRig.CommonData.DTimCreation),
-                DateTimeLastChange = StringHelpers.ToDateTime(witsmlRig.CommonData.DTimLastChange),
-                ItemState = witsmlRig.CommonData.ItemState,
-                Name = witsmlRig.Name,
+                AirGap = witsmlRig.AirGap == null ? null : new LengthMeasure { Uom = witsmlRig.AirGap.Uom, Value = decimal.Parse(witsmlRig.AirGap.Value) },
+                Approvals = witsmlRig.Approvals,
+                ClassRig = witsmlRig.ClassRig,
+                DTimStartOp = StringHelpers.ToDateTime(witsmlRig.DTimStartOp),
+                DTimEndOp = StringHelpers.ToDateTime(witsmlRig.DTimEndOp),
+                EmailAddress = witsmlRig.EmailAddress,
+                FaxNumber = witsmlRig.FaxNumber,
+                IsOffshore = bool.Parse(witsmlRig.IsOffshoreText),
```

Comment:
sufficient to set equal to "witsmlRig.IsOffshore"

Reply:
isOffshore is a bool so isOffshoreText  has to be parsed, what do you think?